src/s3Accounting/components/ComparisonTab.js

```diff
@@ -0,0 +1,381 @@
+import {
+  Container,
+  Dropdown,
+  Grid,
+  Header,
+  Icon,
+  Message,
+  Placeholder,
+  Segment,
+} from 'semantic-ui-react';
+import {FaEquals, FaExchangeAlt, FaGhost, FaQuestion} from 'react-icons/fa';
+import React, {useState} from 'react';
+
+import {ALL_FILE_AUDITS} from '../queries';
+import {Chart} from 'react-charts';
+import FileUploadTable from './FileUploadTable';
+import {dateTime} from '../../common/dateUtils';
+import {useQuery} from '@apollo/client';
+
+const ComparisonTab = ({match, loading, error, data}) => {
+  const [tab, setTab] = useState('matched');
+  const [dropdown, setDropdown] = useState('count_by_data_type');
+  const [pages, setPages] = useState([{first: 10}]);
+
+  const storageAnalyses =
+    data && data.allStorageAnalyses.edges.length > 0
+      ? data.allStorageAnalyses.edges[0].node
+      : {};
+
+  const {
+    loading: fileAuditsLoading,
+    error: fileAuditsError,
+    data: fileAuditsData,
+    refetch: fileAuditsRefetch,
+    fetchMore,
+  } = useQuery(ALL_FILE_AUDITS, {
+    variables: {
+      storageAnalysis: storageAnalyses.id,
+      result: tab,
+      ...pages[0],
+    },
+  });
+
+  const audit = storageAnalyses.stats
+    ? JSON.parse(storageAnalyses.stats).audit
+    : {};
+  console.log(storageAnalyses.stats && JSON.parse(storageAnalyses.stats));
+  const tabContent = {
+    differ: {
+      title: ' S3 locations that have a different file than expected',
+      description:
+        'This metric is computed by counting the urls in the upload manifest(s) that match urls in the S3 inventory, but have a different file hash in the upload manfiest(s) vs the S3 inventory. These represent locations in S3 where file content changed or got overwritten.',
+      color: '#db2828',
+    },
+    matched: {
+      title: ' files that were expected but not found in S3',
+      description:
+        'This metric is computed by counting the file hashes in the upload manifest(s) that were not found in the S3 inventory. These represent file blobs that do not exist anywhere in S3 at the time of scan.',
+      color: '#21ba45',
+    },
+    missing: {
+      title: ' files in S3 with different content than expected',
+      description:
+        'This metric is computed by counting the urls in the upload manifest(s) whose hash value is different than the hash value in the S3 inventory. These items could indicate corrupted uploads or S3 objects that have been overwritten with different content since the time of upload.',
```

Comment:
```suggestion
        'This metric is computed by counting the hashes in the upload manifest(s) that were not found in the S3 inventory. These items could indicate corrupted uploads or S3 objects that have been overwritten with different content since the time of upload.',
```